src/main/java/com/google/api/codegen/transformer/csharp/CSharpGapicSnippetsTransformer.java

```diff
@@ -134,14 +134,49 @@ private SnippetsFileView generateSnippets(GapicInterfaceContext context) {
       } else if (methodConfig.isPageStreaming()) {
         if (methodConfig.isFlattening()) {
           ImmutableList<FlatteningConfig> flatteningGroups = methodConfig.getFlatteningConfigs();
+          // Find flattenings that have ambiguous parameters, and mark them to use named parameters.
```

Comment:
Nit: named arguments, not named parameters. (Including variable name, method names etc.)
I know it's a bit of a nit, but I'd rather we got it right from the start...

Reply:
All changed.